polars/polars-core/Cargo.toml

```diff
@@ -148,6 +148,7 @@ docs-selection = [
 ]
 
 [dependencies]
+wasm-timer = "0.2.5"
```

Comment:
Can this be made an optional dependency? It should only be used when compiled for wasm.